lib/kitchen/driver/oci.rb

```diff
@@ -380,14 +380,18 @@ def mime_parts(boundary)
       end
 
       def user_data
```

Comment:
Could you change this to `def user_data # rubocop:disable Metrics/MethodLength` to prevent Rubocop errors.

Reply:
Done. Want me to squash and rewrite to a single commit?